package.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "sa-map-editor",
+  "version": "0.1.0",
+  "description": "A tool to generate Twilight Imperium Maps for Shatterd Ascension",
+  "private": true,
+  "directories": {
+    "lib": "lib"
+  },
+  "scripts": {
+    "build": "node scripts/pack.mjs",
+    "dev": "serve dist",
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/cayrusa/SA-map-editor.git"
+  },
+  "keywords": [
+    "TI",
+    "TI3",
+    "SA"
+  ],
+  "author": "cayrusa",
+  "contributors": [
+    "lokimidgard",
+    "Tzeen"
+  ],
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/cayrusa/SA-map-editor/issues"
+  },
+  "homepage": "https://github.com/cayrusa/SA-map-editor#readme",
+  "dependencies": {
+    "createjs": "^1.0.1",
+    "easeljs": "^1.0.2",
+    "jquery": "2.1.0",
+    "preloadjs": "^1.0.1",
+    "tweenjs": "^1.0.2",
+    "uglify-js": "^3.14.5"
+  },
```

Comment:
Looking at easeljs, preloadjs and tweenjs, the versions here are superior to the versions of the files in the lib folder (easeljs-0.8.0.min.js, preloadjs-0.6.0.min.js and tweenjs-0.6.0.min.js).
Does that mean these three files in the lib folder are no longer necessary because npm will automatically download these libraries with the versions defined in this file?

Reply:
The site uses still the old js files.

I thought to move the dependencys to the packge.json file, and using npm to resolve those. But it would not be a simple replacement, and my JavaScript expierience is also limited especialy pre ES6.

And while I would like to change it I havn't yet the time to change this....

It can just be removed and the site will still work.